otj-metrics-core/src/main/java/com/opentable/metrics/graphite/GraphiteSenderWrapper.java

```diff
@@ -120,7 +119,7 @@ public synchronized int getFailures() {
     private synchronized Graphite maybeRecycle() throws IOException {
         if (needsReconnectPeriodic()) { //NOPMD
             // Spin up new one
-            Graphite newGraphite = new Graphite(address.get());
```

Comment:
This technically didn't need to change. The supplier is going to force re-resolution every time.